### ISSUE TITLE
rawstudio failed to build with GCC6 or we need update plugin dcraw

### ISSUE THREAD

**Issue by sergiomb2** [rawstudio/rawstudio#19]
Hello , rawstudio failed to build with GCC6 in next Fedora version (24) we need update dcraw , after update  plugins/load-dcraw/ with latest version of ufraw (0.22) it builds again .

Where is my last commit 
https://github.com/sergiomb2/rawstudio/commit/d34809ceb41bcd3af15107a3eb56b1345f8af741

it is working,  the patch is very big (1.5 Mega bytes !) I need to copy from ufraw this files:

+# dcraw.cc
+# dcraw.h
+# dcraw_api.cc
+# dcraw_api.h
+# dcraw_indi.c
+# nikon_curve.c
+# nikon_curve.h
+# uf_glib.h
+# uf_progress.h
+# ufobject.cc
+# ufobject.h
+# ufraw-batch.c
+# ufraw.h
+# ufraw_colorspaces.c
+# ufraw_colorspaces.h
+# ufraw_conf.c
+# ufraw_developer.c
+# ufraw_embedded.c
+# ufraw_exiv2.cc
+# ufraw_message.c
+# ufraw_routines.c
+# ufraw_settings.cc
+# ufraw_ufraw.c
+# ufraw_writer.c
+# wb_presets.c

**Comment by sergiomb2** [comment]
rawstudio 
```
/usr/lib64/rawstudio/plugins/load_dcraw.so: undefined symbol: ufraw_progress
**
ERROR:rs-plugin-manager.c:56:rs_plugin_manager_load_all_plugins: assertion failed: (g_type_module_use(G_TYPE_MODULE(plugin)))

```

I need to check this ...

**Comment by abrander** [comment]
Is there any problems besides dcraw? I'm seriously considering removing dcraw and relying solely on RawSpeed.

**Comment by sergiomb2** [comment]
For me no problem at all. Apart that doesn't compile with gcc6, though that would be nice have it updated , but if is not used , remove it, for me,  is the best solution .
Thanks.

**Comment by abrander** [comment]
This should be fixed by #33.
